views/incorrect-information/wrong-ro.html

```diff
@@ -1,23 +1,103 @@
+                  <p>You will need to
+                    <a href="{{changeRoaUrl}}" target="_blank" rel="noopener noreferrer" data-event-id="update-roa-link">update the registered office address (opens in a new tab)</a>
+                    before you can file a confirmation statement.</p>
+                </li>
+              </ol>
+            </div>
+          </li>
+
+          <li class="app-step-nav__step js-step" id="return-to-this=service">
+            <div class="app-step-nav__header js-toggle-panel" data-position="2">
+              <h2 class="app-step-nav__title">
+                <span class="app-step-nav__circle app-step-nav__circle--number">
+                  <span class="app-step-nav__circle-inner">
+                    <span class="app-step-nav__circle-background">
+                      <span class="govuk-visually-hidden">Step</span> 2
+                    </span>
+                  </span>
+                </span>
+                <span class="js-step-title">
+                  <button class="app-step-nav__button app-step-nav__button--title js-step-title-button" aria-expanded="false" aria-controls="step-panel-return-to-this-service"/>
+                </span>
+                <span class="js-step-title-text">
+                  Return to this service
+                </span>
+              </h2>
+            </div>
+            <div>
+              <ol class="app-step-nav__list " data-length="1">
+                <li class="app-step-nav__list-item js-list-item ">
+                  <p>Once you have updated the registered office address, you will need to close that tab to return to this screen.</p>
+                </li>
+              </ol>
+            </div>
+          </li>
+
+          <li class="app-step-nav__step js-step" id="complete-cs">
+            <div class="app-step-nav__header js-toggle-panel" data-position="3">
+              <h2 class="app-step-nav__title">
+                <span class="app-step-nav__circle app-step-nav__circle--number">
+                  <span class="app-step-nav__circle-inner">
+                    <span class="app-step-nav__circle-background">
+                      <span class="govuk-visually-hidden">Step</span> 3
+                    </span>
+                  </span>
+                </span>
+                <span class="js-step-title">
+                  <button class="app-step-nav__button app-step-nav__button--title js-step-title-button" aria-expanded="false" aria-controls="step-panel-complete-confirmation-statement"/>
+                </span>
+                <span class="js-step-title-text">
+                  Complete your confirmation statement
+                </span>
+              </h2>
+            </div>
+            <div>
+              <p>Return to the <a href="{{changeRoaUrl}}" target="_blank" rel="noopener noreferrer" data-event-id="update-roa-link">Registered office address</a> section and confirm that an update has been submitted.</p>
```

Comment:
this link in the prototype takes you to the registered office address screen, this link on this branch crashes.

Reply:
I have raised a bug to deal with this: https://companieshouse.atlassian.net/browse/NCS-851 it is at the top of the backlog don't worry about this now we have the same problem on main.

Reply:
this is a different link, the prototype takes you to the registered office address screen in the task list for this link